src/EditorFeatures/Core/InheritanceMargin/AbstractInheritanceMarginService.cs

```diff
@@ -33,56 +34,59 @@ internal abstract class AbstractInheritanceMarginService : IInheritanceMarginSer
             TextSpan spanToSearch,
             CancellationToken cancellationToken)
         {
-            var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-            var allDeclarationNodes = GetMembers(root.DescendantNodes(spanToSearch));
-            if (allDeclarationNodes.IsEmpty)
+            using (Logger.LogBlock(FunctionId.InheritanceMargin_GetInheritanceMemberItems, cancellationToken))
```

Comment:
Note that this telemetry, by default, is not uploaded. It is captured for performance traces

Reply:
: ) Yup, as we synced offline, i change the log level here